AssignmentEleven/AssignmentEleven/ViewControllers/AddDetailsViewController.swift

```diff
@@ -0,0 +1,139 @@
+//
+//  AddDetailsViewController.swift
+//  AssignmentEleven
+//
+//  Created by weboniselab1 on 21/08/2015.
+//  Copyright (c) 2015 weboniselab. All rights reserved.
+//
+
+import Foundation
+import UIKit
+import CoreData
+
+class AddDetailsViewController : UIViewController {
+    var isCalledFromPersonDetails : Bool!
+    var localDictionary : Dictionary<String,AnyObject>!
+    
+    let keysOfPersonObject :[String] = ["firstName" , "contactNumber"]
+    let keysOfOrganisation : [String] = ["organisationName" , "location"]
+    var dummyNsmanagedObject : NSManagedObject!
+    
+    @IBOutlet weak var labelFirstNameOrOrg: UILabel!
+    
+    @IBOutlet weak var labelPhoneNumOrLocation: UILabel!
+    
+    @IBOutlet weak var textFiledFirstNameOrOrg: UITextField!
+    
+    @IBOutlet weak var textFiledContactNumOrLoc: UITextField!
+    
+    @IBOutlet weak var customNavigationItem: UINavigationItem!
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        self.navigationController?.navigationBarHidden = true
+        var b = UIBarButtonItem(title: "Cancel", style: .Plain, target: self, action: "backButtonActionHandler")
+        customNavigationItem.leftBarButtonItem = b
+        initiateLabelValues()
+        
+    }
+    
+    override func didReceiveMemoryWarning() {
+        super.didReceiveMemoryWarning()
+        
+        
+    }
+    
+    
+    
+    
+    //initiate labels
+    
+    func initiateLabelValues(){
+        if(isCalledFromPersonDetails == true){
+            labelFirstNameOrOrg.text = "FirstName :"
+            labelPhoneNumOrLocation.text = "Contact Number :"
+        }
+            
+        else{
+            labelFirstNameOrOrg.text = "Organisation Name :"
+            labelPhoneNumOrLocation.text = "Location :"
+        }
+        
+    }
+    
+    
+    @IBAction func buttonSaveActionHandler(sender: UIBarButtonItem) {
+        var entity : String!
+        var messageForAlertView : String!
+        //call makedictionary to initialize localdictionary
+        if(textFiledFirstNameOrOrg.text.isEmpty){
+            if(isCalledFromPersonDetails==true){
+                messageForAlertView = "Enter Name"
+            }
+            else{
+                messageForAlertView = "Enter Organisation Name"
+            }
+            
+            alertMessage("Invalid!!", message: messageForAlertView)
+        }
+        else if(textFiledContactNumOrLoc.text.isEmpty){
+            if(isCalledFromPersonDetails==true){
```

Comment:
You can write that code as 
if(isCalledFromPersonDetails) {
}

No need to compare with true value, by default it will check for true in if block.